packages/web-client/tests/integration/components/degraded-service-banner-test.ts

```diff
@@ -0,0 +1,194 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, waitFor } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { Response as MirageResponse } from 'ember-cli-mirage';
+import { MirageTestContext } from 'ember-cli-mirage/test-support';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+interface Context extends MirageTestContext {}
+
+module('Integration | Component | degraded-service-banner', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  test('It doesn not display a banner when there are no incidents', async function (this: Context, assert) {
+    this.server.get(
+      'https://status.cardstack.com/api/v2/incidents/unresolved.json',
```

Comment:
Not that this is likely to change, but would it make sense to use the host from the config in the tests too?

Reply:
Yup! Will clean it up.